src/js/InstitutionStatus.jsx

```diff
@@ -28,7 +28,7 @@ var InstitutionStatus = React.createClass({
     var submission = this.props.institution.currentSubmission;
     var submissionRoute = submission === 0 ? 1 : submission;
     var appLink = '/' + id + '/' + period + '/' + submissionRoute;
-    var statusLink = <Link to={appLink}>View filing status</Link>
+    var statusLink = <Link className="statusLink" to={appLink}>View filing status</Link>
```

Comment:
I can see what you mean here, but imo, the class is a good choice here. I'd say since the objective is purely stylistic (more whitespace here), using a class with styling information is truer to the intent than wrapping the element.